tests/ert/unit_tests/workflow_runner/test_workflow_runner.py

```diff
@@ -103,23 +103,28 @@ def test_run_internal_script():
     assert result == -1
 
 
-@pytest.mark.usefixtures("use_tmpdir")
 @pytest.mark.parametrize(
     "config, expected_result",
     [
         (["INTERNAL FALSE"], "FALSE has no effect"),
-        (["SCRIPT scripy.py"], "SCRIPT has no effect"),
-        (["SCRIPT scripy.py", "INTERNAL TRUE"], "SCRIPT and INTERNAL"),
+        (["SCRIPT script.py"], "SCRIPT has no effect"),
```

Comment:
:smile:

Reply:
I used to mock everything, so this was not relevant, only spotted my mistake when I actually had to use it 😅